src/nemo-connectivity/mobiledataconnection.cpp

```diff
@@ -159,6 +160,53 @@ void MobileDataConnectionPrivate::updateServiceProviderName()
     }
 }
 
+void MobileDataConnectionPrivate::updateTechnology()
+{
+    NetworkTechnology *newTech;
+    QString technologyPath = QString("cellular");
```

Comment:
I think QStringLiteral() would make sense here.

Reply:
Yes it does. Changed with 0f33fcf8e5d8515be88142ca1e5780d754028fd6